village/src/components/SmurfForm.js

```diff
@@ -10,10 +10,15 @@ class SmurfForm extends Component {
     };
   }
 
-  addSmurf = event => {
+  addHelper = event => {
     event.preventDefault();
     // add code to create the smurf using the api
-
+    let newSmurf = {
+      name: this.state.name,
+      age: this.state.age,
+      height: this.state.height
+    }
+    this.props.addSmurf(newSmurf)
```

Comment:
Good job in knowing that you need to pass state up instead of doing `axios.post` inside this component.

Refactoring opportunity:
We can pass `this.state` instead of `newSmurf` bec they have the same shape 
Lines 16-20 can be removed
```suggestion
    this.props.addSmurf(this.state)
```